code/modules/mob_spawn/corpses/mining_corpses.dm

```diff
@@ -207,5 +207,5 @@
 	suit_store = /obj/item/tome
 	l_pocket = /obj/item/melee/cultblade/dagger
 	back = /obj/item/storage/backpack/cultpack
-	glasses = /obj/item/clothing/glasses/hud/health/night/cultblind
+	r_pocket = /obj/item/clothing/glasses/hud/health/night/cultblind
```

Comment:
wut

Reply:
cult blind glasses literally can not be equipped on corpses along with a lot of other gear. the mob spawn was originally dodging the unit tests by overriding outfit slots

Reply:
haha